src/components/content/credential/AddCredential.tsx

```diff
@@ -73,27 +98,24 @@ function AddCredential({
         }
         void CredentialsManagementService.getCredentialCapabilitiesByCsp(currentCsp, currentType)
             .then((resp) => {
+                const names: string[] = [];
+                let credentialInfo: CredentialVariables[] = [];
                 if (resp.length > 0) {
-                    const credentialVariables: CredentialVariables = resp[0] as CredentialVariables;
-                    const credentialVariableList: CredentialVariable[] = [];
-                    credentialVariables.variables.forEach((credentialVariable) => {
-                        credentialVariable.value = '';
-                        credentialVariableList.push(credentialVariable);
+                    // eslint-disable-next-line array-callback-return
```

Comment:
pleas check how to avoid this exception instead of suppressing.  Using a `forEach` should mostly avoid this warning from ES Lint.